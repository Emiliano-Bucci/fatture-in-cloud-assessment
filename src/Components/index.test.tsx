import React from "react";
import { render } from "@testing-library/react";
import { Component } from "components";

describe("test", () => {
  test("test", () => {
    const { container } = render(<Component />);

    expect(container).toMatchSnapshot();
  });
});
