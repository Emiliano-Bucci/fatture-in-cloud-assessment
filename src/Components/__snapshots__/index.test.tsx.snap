// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test test 1`] = `
.emotion-0 {
  background: red;
}

<div>
  <div
    class="emotion-0"
  >
    Component
  </div>
</div>
`;
